app/views/components/_markdown-editor.html.erb

```diff
@@ -1,17 +1,26 @@
+<%
+  textarea||= {}
+  edit_button_text||= 'Edit'
+  preview_button_text||= 'Preview'
+%>
 <div class="app-c-markdown-editor" data-module="markdown-editor">
+  <% if textarea[:label] %>
   <%= render "govuk_publishing_components/components/label", {
```

Comment:
Could you indent these, so it's easier to follow the start/end of the blocks?

Reply:
Indented